documentation/endpoints/signups.md

```diff
@@ -0,0 +1,20 @@
+# Signups
+
+Post an existing Signup to upsert Gambit cache for given Signup, and send a signup confirmation message via chatbot to the Signup.user per Signup.source.
+
+```
+POST /v1/signups
+```
+
+**Headers**
+
+Name | Type | Description
+--- | --- | ---
+`x-gambit-api-key` | `string` | **Required.** 
+ 
+**Input**
+
+Name | Type | Description
+--- | --- | ---
+`id` | `number` |  **Required.**  Signup ID to update Gambit cache for
```

Comment:
@sergii-tkachenko Also ok to pass `id` and `source` to POST `signups` now (seemed redundant to keep the params as `signup_id` and `signup_source` for new `signups` endpoint.